packages/expo-apple-authentication/src/SignInWithAppleButton.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import {
+  ExpoSignInWithAppleButtonSignInWhite,
+  ExpoSignInWithAppleButtonSignInWhiteOutline,
+  ExpoSignInWithAppleButtonSignInBlack,
+  ExpoSignInWithAppleButtonContinueWhite,
+  ExpoSignInWithAppleButtonContinueWhiteOutline,
+  ExpoSignInWithAppleButtonContinueBlack,
+} from './ExpoSignInWithAppleButtonNativeViews';
+
+import {
+  ButtonProps,
+  ButtonStyle,
+  ButtonType,
+} from './AppleAuthentication.types';
+
+/**
+ * This component displays the "Sign In with Apple" button on your screen.
+ * The App Store Guidelines require you to use this component to start the sign in process instead of a custom button.
+ * You can customise the design of the button using the properties.
+ * You should start the sign in process when the `onPress` property is called.
+ *
+ * You should only attempt to render this if `SignInWithApple.isAvailableAsync()` resolves to true.
+ * This component will render nothing if it is not available and you will get a warning in `__DEV__ === true`.
+ *
+ * The properties of this component extends from `View`, however, you should not attempt to restyle the background color
+ * or border radius with the style property.
```

Comment:
we expose a cornerRadius property -- how does this interplay with style.borderRadius? This is the kind of detail we need to communicate to developers.